RecoTracker/TrackProducer/src/DAFTrackProducerAlgorithm.cc

```diff
@@ -210,16 +210,12 @@ bool DAFTrackProducerAlgorithm::buildTrack (const Trajectory vtraj,
 					    float ndof,
 					    const reco::BeamSpot& bs) const
 {
-  //variable declarations
-  reco::Track * theTrack;
-  Trajectory * theTraj; 
-      
   LogDebug("DAFTrackProducerAlgorithm") <<" BUILDER " << std::endl;;
   TrajectoryStateOnSurface innertsos;
   
   if ( vtraj.isValid() ){
 
-    theTraj = new Trajectory( vtraj );
+    std::unique_ptr<Trajectory> theTraj(new Trajectory( vtraj ));
```

Comment:
I hoped there would not be merged conflicts with #7981, but taking a closer look now there will be (trivial to solve but probably not done automatically). It is probably easiest if #7981 gets merged first and then I fix this one.